doc/oemof_solph.rst

```diff
@@ -698,6 +701,88 @@ Link.
 .. note:: See the :py:class:`~oemof.solph.custom.Link` class for all parameters and the mathematical background.
 
 
+.. _oemof_solph_custom_sinkdsm_label:
+
+
+SinkDSM (custom)
+^^^^^^^^^^^^^^^^
+
+:class:`~oemof.solph.custom.SinkDSM` can used to represent flexibility in a demand time series.
+Elasticity of the demand is described by upper (:attr:`~oemof.solph.custom.SinkDSM.capacity_up`) and lower (:attr:`~oemof.solph.custom.SinkDSM.capacity_down`) bounds where within the demand is allowed to vary.
+Upwards shifted demand is then balanced with downwards shifted demand.
+
```

Comment:
Could you please add a sentence about the current state of the component as you did in the docstring.

Reply:
I added the same note at the end of the section about SinkDSM in the Note box